docs/_SUMMARY.md

```diff
@@ -1,7 +1,13 @@
+Please join #cdd-nim-anywhere slack channel if you are a internal user, open an issue if you are external for any question and feedback.
```

Comment:
It would be better to have all of this in the actual document before the quick start and leave the summary very short. That way, the diagram stays close to the top. Up to you